src/components/Home.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
+import { Button } from 'react-bootstrap';
 
 const Home = () => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="text-center">
+      <section>
+        <h2>{"{ CYF Quiz }"}</h2>
```

Comment:
This should be a `h1` as it is the main heading of the page. Also, I don't think we need to have `{` and `}` as it looks a bit unusual

Reply:
yeah, It was `h1` but I thought to change the font size.
removed the `{...}`.